src/encoded/static/components/biosample_type.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as globals from './globals';
+import { Breadcrumbs } from './navigation';
+import { DbxrefList } from './dbxref';
+import { PickerActions } from './search';
+import { auditDecor } from './audit';
+import pubReferenceList from './reference';
+import BiosampleTermId from './biosample';
+
+
+const BiosampleTypeComponenet = (props, reactContext) => {
+    const context = props.context;
+    const itemClass = globals.itemClass(context, 'view-detail key-value');
+
+    // Set up breadcrumbs
+    const crumbs = [
+        { id: 'BiosampleTypes' },
+        { id: context.classification, query: `classification=${context.classification}`, tip: context.classification },
+        { id: context.term_name, query: `term_name=${context.term_name}`, tip: context.term_name },
+    ];
+
+    // Get a list of reference links, if any
+    const references = pubReferenceList(context.references);
+
+    return (
+        <div className={itemClass}>
+            <header className="row">
+                <div className="col-sm-12">
+                    <Breadcrumbs root="/search/?type=Biosample" crumbs={crumbs} />
```

Comment:
`root` should be "/search/?type=BiosampleType" I think. I don’t think breadcrumbs will show any results with "Biosample" here.

Reply:
Right.